app/jinja_filters.py

```diff
@@ -288,9 +289,11 @@ def __init__(self, option, index, answer):
             # the 'pre-' prefix is added to the attributes here so that html minification
             # doesn't mess with the attribute contents (the 'pre-' is removed during minification).
             # see https://htmlmin.readthedocs.io/en/latest/quickstart.html
+            attribute_key = "pre-" if settings.EQ_ENABLE_HTML_MINIFY else ""
```

Comment:
Should we not be getting the config from current app config like we do elsewhere instead of accessing in directly?